zwave_js_server/model/node.py

```diff
@@ -241,6 +241,52 @@ async def async_set_value(self, val: Union[Value, str], new_value: Any) -> None:
             }
         )
 
+    async def async_refresh_info(self) -> None:
+        """Send refreshInfo command to Node."""
+        await self.client.async_send_json_message(
+            {
+                "command": "node.refresh_info",
+                "nodeId": self.node_id,
+            }
+        )
+
+    async def async_get_defined_value_ids(self) -> List[Value]:
+        """Send getDefinedValueIDs command to Node."""
+        data = await self.client.async_send_command(
+            {
+                "command": "node.get_defined_value_ids",
+                "nodeId": self.node_id,
+            }
+        )
+        return [
+            Value(self, cast(ValueDataType, valueId))
+            for valueId in list(data["valueIds"])
```

Comment:
Why first convert it to a list? That's not needed

```suggestion
            for valueId in data["valueIds"]
```

Reply:
I got some issues from `mypy` before. I will try it without it ;)